qt/configure

```diff
@@ -172,6 +172,7 @@ addComment "application"
 addInstallDir                           "/bin"
 addInstallFile "backintime-qt"         "/bin" "755"
 addInstallFile "backintime-qt_polkit"  "/bin" "755"
+printf "\tinstall --mode=644 io.github.bit_team.back_in_time.gui.xml /usr/share/metainfo\n"  >> ${MAKEFILE}
```

Comment:
I don't have enough time to check it. I have almost no knowledge about it anyway. I suppose that if you followed "Debian AppStream guidelines" (I suppose https://wiki.debian.org/AppStream/Guidelines) is ok.
Automatic check (https://appstream.debian.org/) is done for packages in repository, so will be checked on next version including it will be uploaded.

Reply:
Seems to be `/usr/share/app-info/xmls` on Arch but I have not validated that info yet.